README.md

```diff
@@ -92,13 +92,11 @@ Now you should be able to run Julia like this:
 
     julia
 
-If everything works correctly, you will see a Julia banner and an interactive prompt into which you can enter expressions for evaluation. (Errors related to libraries might be caused by old, incompatible libraries sitting around in your PATH. In that case, try moving the `julia` directory earlier in the PATH).
+If everything works correctly, you will see a Julia banner and an interactive prompt into which you can enter expressions for evaluation. (Errors related to libraries might be caused by old, incompatible libraries sitting around in your PATH. In this case, try moving the `julia` directory earlier in the PATH).
 
-Your first test of Julia should be to determine whether your
-build is working properly. From the UNIX/Windows command prompt inside
+Your first test of Julia determines whether your build is working properly. From the UNIX/Windows command prompt inside
 the `julia` source directory, type `make testall`. You should see output
-that lists a series of tests being run; if they complete without
-error, you should be in good shape to start using Julia.
+that lists a series of running tests; if they complete without error Julia can be started.
```

Comment:
The verb here was "use", so the direct translation into the passive voice would be "the use of Julia can start". That's rather awkward though, since the use of passive voice is not required in the readme. Although, the current wording is quite informal, which might not be ideal either. Maybe you could rewrite this to be more precise, but keep it in the active voice?

Optionally, another useful addition here might be to mention "If something isn't working right, please file a [bug report](https://github.com/JuliaLang/julia/issues/new)! That may be the only way that the developers find out that there is a bug and who it affects." (ok, that's not my favorite sentence structure either, maybe you could help me with word-smithing it also 🙂 )